environments.json

```diff
@@ -2043,9 +2043,24 @@
       "non-standard"
     ]
   },
-  "phantom": {
+  "phantom2": {
     "full": "PhantomJS 2.0",
     "short": "PJS",
+    "link": "https://github.com/ariya/phantomjs",
+    "release": "2015-01-24",
+    "retire": "2016-01-24",
+    "obsolete": "very",
+    "platformtype": "engine",
+    "equals": "safari6",
+    "needs_annex_b": true
+  },
+  "phantom2_1": {
+    "full": "PhantomJS 2.1",
+    "short": "PJS",
+    "link": "https://github.com/ariya/phantomjs",
+    "release": "2016-01-24",
+    "retire": "2018-06-02",
```

Comment:
the linked date is March 2nd

Reply:
Yep, the issue is from March 3rd, but there he says that  he's going to archive "soon".  the actual commit was on June 2nd: https://github.com/ariya/phantomjs/commit/466ae257a91a88200976e81f0668dc309329e518